iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.inl

```diff
@@ -41,17 +42,26 @@ inline typename ChunkSender<ChunkSenderDataType>::MemberType_t* ChunkSender<Chun
 
 template <typename ChunkSenderDataType>
 inline cxx::expected<mepoo::ChunkHeader*, AllocationError>
-ChunkSender<ChunkSenderDataType>::tryAllocate(const uint32_t payloadSize, const UniquePortId originId) noexcept
+ChunkSender<ChunkSenderDataType>::tryAllocate(const uint32_t payloadSize,
+                                              const UniquePortId originId,
+                                              const uint32_t payloadAlignment,
+                                              const uint32_t customHeaderSize,
+                                              const uint32_t customHeaderAlignment) noexcept
 {
     // use the chunk stored in m_lastChunk if there is one, there is no other owner and the new payload still fits in it
-    const uint32_t neededChunkSize = getMembers()->m_memoryMgr->sizeWithChunkHeaderStruct(payloadSize);
+    const uint32_t neededChunkSize = getMembers()->m_memoryMgr->requiredChunkSize(
+        payloadSize, payloadAlignment, customHeaderSize, customHeaderAlignment);
 
```

Comment:
I think this whole code would be a little bit more understandable with
`auto & lastChunk = getMembers()->m_lastChunk;`
But it is not a must, just a suggestion.